test/concurrent/ThreadPoolTest.ooc

```diff
@@ -27,6 +27,11 @@ ThreadPoolTest: class extends Fixture {
 			expect(promise3 wait())
 			expect(promise4 wait(), is false)
 			expect(promise5 wait(), is true)
+			expect(promise1 _state as Int, is equal to(_PromiseState Finished as Int))
+			expect(promise2 _state as Int, is equal to(_PromiseState Finished as Int))
+			expect(promise3 _state as Int, is equal to(_PromiseState Finished as Int))
+			expect(promise4 _state as Int, is equal to(_PromiseState Cancelled as Int))
+			expect(promise5 _state as Int, is equal to(_PromiseState Finished as Int))
```

Comment:
Yeah of course. The question if these should pass or not remains though. Is it a bug that they all fail or is it intended that way?